VAMobile/src/screens/HealthScreen/LabsAndTests/LabsAndTestsDetails/LabsAndTestsDetailsScreen.test.tsx

```diff
@@ -0,0 +1,135 @@
+      sampleSite: 'TESTING BONE MARROW',
+      location: 'VA TEST LAB',
+      orderedBy: 'Provider Name',
+    },
+  }
+  const chemHemLabsAndTests = {
+    id: 'I2-2BCP5BAI6N7NQSAPSVIJ6INQ4A000000',
+    type: 'diagnostic_report',
+    attributes: {
+      display: 'Chemistry/Hematology',
+      testCode: 'CH',
+      dateCompleted: '2018-11-01T15:49:14.000-01:00',
+      encodedData: '',
+      sampleSite: 'TESTING BONE MARROW',
+      location: 'VA TEST LAB',
+      observations: [
+        {
+          testCode: 'GLUCOSE',
+          valueQuantity: '100',
+          referenceRange: '70-100',
+          status: 'final',
+          comment: 'this is a test',
+        },
+        {
+          testCode: 'UREA NITROGEN',
+          valueQuantity: '200 mg/dL',
+          referenceRange: '7 - 18',
+          status: 'pending',
+          comment: '',
+        },
+        {
+          testCode: 'CREATININE',
+          valueQuantity: '5 mg/dL',
+          referenceRange: '0.6 - 1.3',
+          status: 'pending',
+          comment: '',
+        },
+        {
+          testCode: 'SODIUM',
+          valueQuantity: '8 meq/L',
+          referenceRange: '136 - 145',
+          status: 'pending',
+          comment: '',
+        },
+      ],
+    },
+  }
+
+  const initializeTestInstance = (labAndTest: LabsAndTests = defaultLabsAndTests) => {
+    const props = mockNavProps(undefined, undefined, { params: { labOrTest: labAndTest } })
+    return render(<LabsAndTestsDetailsScreen {...props} />)
+  }
+
+  it('base64 decodes the encodedData -- report data', async () => {
+    const dataWithAtob = {
+      ...defaultLabsAndTests,
+    }
+    const { getByTestId } = initializeTestInstance({ ...dataWithAtob })
+    await waitFor(() =>
+      expect(getByTestId('decoded-report').children[0]).toEqual(
+        `Date Spec taken: Nov 01, 2018 15:49  Pathologist:MURTUZA LOKHANDWALADate Spec rec'd: Nov 01, 2018 15:51  Resident: Date  completed: Nov 01, 2018        Accession #: SP 18 5Submitted by: KALAHASTI, VENKATA S   Practitioner:PADMA BODDULURI-------------------------------------------------------------------------------Specimen: BONE MARROW=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--=--Performing Laboratory:Surgical Pathology Report Performed By: CHYSHR TEST LAB2360 E PERSHING BLVD CHEYENNE, FL 82001-5356null`,
+      ),
+    )
+  })
+
+  it('renders the placeholder for encodedData if no encodedData is present', async () => {
+    const sampleData = {
+      ...defaultLabsAndTests,
+      attributes: {
+        ...defaultLabsAndTests.attributes,
+        encodedData: undefined,
+      },
+    }
+    const { getByTestId } = initializeTestInstance({ ...sampleData })
+    await waitFor(() => expect(getByTestId('decoded-report').children[0]).toEqual('None noted'))
```

Comment:
is there a way to confirm the expected text is rendered in the expected field without using subscript